### PR TITLE
page header breadcrumbs

### PR DESCRIPTION
**Pull request by TaraTodd123**
#### Checklist
* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licenced under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-3494

#### Description of Proposed Changes

- Update page header breadcrumbs color and hover as per feedback from Roland

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects-2_page-header-breadcrumbs

### CODE REVIEW

File: src/components/page-header/page-header.component.less [hunk 1]
```diff
@@ -77,6 +77,19 @@ ux-page-header {
...
+
+                .breadcrumb > li a:hover {
+                    color: @white;
+                }
+
+                .breadcrumb {
+                    li {
+                        a,
+                        span {
+                            color: #00a7a2;
```
Comment (ashley-hunter) [inline src/components/page-header/page-header.component.less]: Can we use `@primary` instead of the hard coded value here?

**Comment by TaraTodd123** [comment]
Updated

**Comment by ashley-hunter** [comment]
Hi Tara,

Should the chevron between breadcrumbs also be changed in some way? It is really hard to see currently:

image

**Comment by ashley-hunter** [comment]
Looks like the build is failing, are there screenshots that need updated?

**Comment by TaraTodd123** [comment]
No screenshots need updated for this component. Seems to be failing on random screenshot tests, I'm running the build again.

**Comment by TaraTodd123** [comment]
Passed now

**Review by ashley-hunter — APPROVED** [review]
Looks good to me, thanks

**Review by donna-marijne — APPROVED** [review]
Looks good to me, thank you.

**Pull request merged**